tests/package/Makefile

```diff
@@ -5,56 +5,47 @@ default: all
 
 clean:
 	rm -rf target/*
-	docker rmi -f marathon-package-test:debian-systemd || echo "Couldn't remove debian-systemd"
-	docker rmi -f marathon-package-test:debian-systemv || echo "Couldn't remove debian-systemv"
+	docker rmi -f marathon-package-test:debian8 || echo "Couldn't remove debian-systemd"
+	docker rmi -f marathon-package-test:debian9 || echo "Couldn't remove debian-systemv"
```

Comment:
This should be updated to remove the ubuntu images, too, right?

Also where is debian9 built?